aspnetcore/blazor/common/samples/3.x/BlazorWebAssemblySample/Program.cs

```diff
@@ -15,7 +15,7 @@ public static async Task Main(string[] args)
             var builder = WebAssemblyHostBuilder.CreateDefault(args);
             builder.RootComponents.Add<App>("app");
 
-            builder.Services.AddSingleton(new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+            builder.Services.AddScoped(new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
```

Comment:
@guardrex This change means the code will no longer compile because `AddScoped` can't be given a singleton instance.

Reply:
That's true, I overlooked this. It has to be `builder.Services.AddScoped<HttpClient>(sp => new HttpClient...`

Reply:
Find-'n-replace ......... *It's GrEaT!* :smile: ........... **_Until it isn't!_** 🙈

I'll patch these now and take the update live immediately.